recipes/syft/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "syft" %}
+{% set version = "0.2.2a2" %}
+
+package:
+  name: {{ syft }}
```

Comment:
Please use:
```
name: {{ name|lower }}
version: {{ version }}
```
instead.

Reply:
You can read more about using variables here, https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#package-section.